tests/mock_crust/churn.rs

```diff
@@ -181,20 +219,26 @@ fn request_during_churn_node_to_group() {
         let data_id = data.identifier();
         let message_id = MessageId::new();
 
+        let except_index = added_index.unwrap_or(nodes.len());
+        let quorum_before = quorum(nodes.iter()
+            .enumerate()
+            .filter(|&(i, n)|
+                i != except_index && n.routing_table().is_recipient(&dst.to_destination()))
```

Comment:
we can't.
the `target_group` calls each node's `routing_table()` function, and at that moment, the new node is not even in `node` state, which will cause the previous `unreachable` panic.